terraform_alert_config/import_adapter/README.md

```diff
@@ -0,0 +1,69 @@
+# Terraform Alerting Config
+This document describes the example DICOM Import adapter terraform configuration. The terraform configuration deploys a DICOM Import adapter to Google Kubernetes Engine, creates a Cloud Healthcare dataset and dicom store, and configures monitoring to alert on the health of the adapter.
+
+## Prepare workspace
+The following software must be installed to use this configuration:
+1. [gcloud](https://cloud.google.com/sdk/docs/install)
+2. [Terraform](https://www.terraform.io/downloads.html)
+3. [Kubernetes](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+# Description
+Terraform configuration for deploying alerting on DICOM Import Adapter health.
+
+The import adapter alerts on the following error events:
+* CSTORE_ERROR
+* CFIND_ERROR
+* CFIND_QIDORS_ERROR
+* CMOVE_ERROR
+* CMOVE_QIDORS_ERROR
+* CMOVE_CSTORE_ERROR
+* COMMITMENT_ERROR
+* COMMITMENT_QIDORS_ERROR
+
+### Notification alerting
+The configuration provides the ability to send notifications to the user in the event of an event, which is defined in the *terraform.tfvar* file.
+
+### Health check
+The configuration provides the ability to control the cluster uptime and send the corresponding event to the monitoring system.
```

Comment:
phrasing edit:
The configuration provides the ability to control
->
The configuration provides the ability to monitor